Analysis/Tasks/PWGLF/NucleiSpectraTask.cxx

```diff
@@ -61,7 +61,7 @@ struct NucleiSpecraTask {
   Filter collisionFilter = nabs(aod::collision::posZ) < cfgCutVertex;
   Filter trackFilter = (nabs(aod::track::eta) < cfgCutEta) && (aod::track::isGlobalTrack == (uint8_t) true);
 
-  using TrackCandidates = soa::Filtered<soa::Join<aod::Tracks, aod::TracksExtra, aod::pidRespTPC, aod::pidRespTOF, aod::pidRespTOFbeta, aod::TrackSelection>>;
+  using TrackCandidates = soa::Filtered<soa::Join<aod::Tracks, aod::TracksExtra, aod::pidRespTPCHe, aod::pidRespTOFHe, aod::TrackSelection>>;
```

Comment:
@lbariogl I now move from the full table to the split one by default, this is transparent for you, on the pidTOF and pidTPC side, i.e. `o2-analysis-pid-tof --pid-he 1`. This is to let you know.